renovate.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", ":dependencyDashboard"],
-  "enabledManagers": ["helm-requirements","gradle-wrapper", "gradle", "nvm", "npm", "terraform"],
+  "extends": [
+    "local>hmcts/.github:renovate-config",
+  "local>hmcts/.github//renovate/automerge-all"],
```

Comment:
If you could just sort format of this like we discussed pls

Reply:
resolved .